Xamarin.Forms.Core/Setter.cs

```diff
@@ -14,6 +14,10 @@ public sealed class Setter : IValueProvider
 	{
 		readonly ConditionalWeakTable<BindableObject, object> _originalValues = new ConditionalWeakTable<BindableObject, object>();
 
+		public BindableObject Target { get; set; }
+
+		public string TargetName { get; set; }
```

Comment:
I find weird that you can use either Target or TargetName, as they do not look mutually exclusive